### DISCUSSION TITLE
Is this from a plugin?

### DISCUSSION THREAD

**Discussion by kls1991** [NeogitOrg/neogit#994] [Q&A] [answered]
The numbers on right-up corner of neogit's showcase picture seem neat. Is it from a neovim plugin, or from terminal emulator, or from window manager?
Sorry for the question which does not relate to neogit a lot, I really like it :)
![259042023-d964cbb4-a557-4e97-ac5b-ea571a001f5c](https://github.com/NeogitOrg/neogit/assets/32974383/731fff7d-92c2-4dfd-94eb-031e45ec8b7b)

**Comment by CKolkey** [comment, marked as answer]
Those are tab pages! The default way to launch Neogit is in a new tab - you probably have a tab bar plugin that shows the them in the top right. Not a neogit thing, but definitely another neovim plugin.

Hope that helps :)
